src/main/resources/db/changelog/constrains.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE financial_transactions ADD CHECK (amount > 0);
```

Comment:
```suggestion
ALTER TABLE financial_transactions ADD CONSTRAINT financial_transaction_check_amount CHECK (amount > 0);
```
Add constraint name.